src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/prestashop_ui_kit.html.twig

```diff
@@ -22,4 +22,107 @@
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  * International Registered Trademark & Property of PrestaShop SA
  *#}
-{% use '@PrestaShop/Admin/TwigTemplateForm/bootstrap_4_horizontal_layout.html.twig' %}
+{% use 'bootstrap_4_horizontal_layout.html.twig' %}
```

Comment:
@eternoendless  As this one is not used because of the wrong path: bootstrap_4_horizontal_layout.html.twig - can we delete this line?